lib/index.js

```diff
@@ -46,6 +46,25 @@ Customerio.prototype.loaded = function() {
 };
 
 /**
+ * Page.
+ *
+ * https://learn.customer.io/api/#apipageview_event
+ * 
+ * @api public
+ * @param {Page} page
+ */
+Customerio.prototype.page = function(page) {
+  var name = page.name();
+  var data = page.properties();
+
+  if (!name) {
+    name = data.url;
```

Comment:
Does this handle if data.url is undefined? The API seems to require it - perhaps an early return? Also, you can use <Facade>.proxy() to pull nested values. :)

Example here: 
var name = page.name() || page.proxy('properties.data.url');
if (!name) return;